### ISSUE TITLE
Better version of Prison Sink

### ISSUE THREAD

**Issue by grandsong** [yooksi/pa-aio-deluxe#2]
After I became mostly AFK, the dev team someday introduced Sink and Mirror. Just when I was examining your changes, I for the first time looked carefully at that thing. 

And It makes me frown.

To begin with, I totally **agree** with you that it is very **unrealistic** to allow **glass** mirror installed in prison.

However, even without the mirror part, I also found problems with the sink part.

1. the sink is just a rectangle. Looks strange since usually sinks are round-corned ones or even nearly ellipses.

2. the tap of the sink is way too tiny and unclear.

3. the tap is not at the center of its side. 

4. the whole thing is way too nasty. It is true that most inmates won't do cleaning very much, but if this logic/reasoning goes further, all things in cells should be as nasty. I don't think it would be a good idea to do so. Maybe a very messy, dirty, unpleasant prison is more realistic, but it would kill the joy for most players including me.

To sum up, the sink part is quite bad. It seems that the dev team was too lazy to design & draw a good picture. 

I suggest we replace it with my "Personal Sink", or something similar. 

Maybe smaller. In the real world, most sinks are at the same size of toilets.

What do you think?

PS: I've found [a web site "shutterstock" which contains many photos of cells](https://www.shutterstock.com/search/prison+cell+sink?image_type=photo)

**Comment by matshou** [comment]
> the sink is just a rectangle. Looks strange since usually sinks are round-corned ones or even nearly ellipses.

I agree we should make a round-cornered version.
 
> the tap of the sink is way too tiny and unclear.
>
> the tap is not at the center of its side.

I was bothered with that as well and tried to move it around but couldn't get it to look good so I gave up, 
however I can take another stab at this as see what I can do.

> the whole thing is way too nasty. It is true that most inmates won't do cleaning very much, but if this logic/reasoning goes further, all things in cells should be as nasty. I don't think it would be a good idea to do so. Maybe a very messy, dirty, unpleasant prison is more realistic, but it would kill the joy for most players including me.

I agree with this as a general rule. The nasty sink version can be used for *nasty* or *dirty* prisons like [this one](https://www.shutterstock.com/image-photo/broken-rusty-dirty-sink-cell-alcatraz-1712686402).

> I suggest we replace it with my "Personal Sink", or something similar.

I think the *Personal Sink* looks way too nice for prison cells. Maybe it fits the higher end (kingpin) cells but I mostly used it to decorate infirmaries, staff rooms and toilets.

> Maybe smaller. In the real world, most sinks are at the same size of toilets.

Agreed, I was already experimenting with re-scaling sinks to see what looks best (see: https://github.com/yooksi/pa-aio-deluxe/commit/02abcf9c1631ac0d415e72dc0a53d9e5c17fb9d0).

**Comment by grandsong** [comment]
Please add your future version of Prison Sink to my latest sprites.png.
After that, if you think it is time, let me know and I will publish this mod on Steam Workshop as an update of mine.
